### ISSUE TITLE
Add metric for propagated transactions size

### ISSUE THREAD

**Issue by Rjected** [paradigmxyz/reth#5846]
### Describe the feature

We currently have the following metrics for transactions:
https://github.com/paradigmxyz/reth/blob/c066aa60af38baa173aaa12d38803ea9062f2703/crates/net/network/src/metrics.rs#L56-L61

The `propagated_transactions` metric is useful, but does not necessarily give us insight into the network load caused by propagation. We could do that by adding a similar metric that counts the size of the propagated transactions in bytes.

**Comment by ameya-deshmukh** [comment]
@Rjected I'd like to pick this up!

**Comment by 1010adigupta** [comment]
would like to work on this with some guidance :)

**Comment by mattsse** [comment]
cool, this will be possible after #6012 , merging shortly

fyi @emhane

**Comment by Rjected** [comment]
@1010adigupta have you had time to work on this? do you need any help?

**Comment by ameya-deshmukh** [comment]
@Rjected I can take this up on the weekend if Adi is cool with it :)

**Comment by Rjected** [comment]
> @Rjected I can take this up on the weekend if Adi is cool with it :)

Sure, if you have time go for it!

**Comment by 1010adigupta** [comment]
> @Rjected I can take this up on the weekend if Adi is cool with it :)

yeah sure you can take this up, was not able to look into it much :)

**Comment by emhane** [comment]
would be really nice if this was grouped by version (eth66 or eth68) and tx type. that can also be a separate issue though.

**Comment by ameya-deshmukh** [comment]
hey @onbjerg! no blockers as such but i'm still working on a wip pr. will updated by late friday/early saturday if that works. thanks :)

**Comment by jenpaff** [comment]
will open new issue with more details
